aten/src/ATen/native/cuda/Indexing.cu

```diff
@@ -1334,8 +1334,18 @@ tensorInfoLegacyIfScalar(cuda::detail::TensorInfo<T, IndexType> ti) {
   return ti;
 }
 
+uint64_t getDefaultMaxThreadsPerBlock() {
```

Comment:
May be use a `constexpr` here, since that should preserve the overhead characteristics before the changes in this PR, when 128 was hardcoded.